### PR TITLE
JWT setup for authentication

### PR DESCRIPTION
**Pull request by roy-widergy**
JWT setup for authentication. Signup, login and logout ok!

Signup ok (response 200):
image

Login ok (response 200):
image

Logout ok (response 204):
image

Wrong login (response 401) 
image

### CODE REVIEW

File: app/helpers/registration_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module RegistrationHelper
```
Comment (GabrielPiccini) [inline app/helpers/registration_helper.rb:1]: Delete this file if you are not using it

File: app/helpers/sessions_controller_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module SessionsControllerHelper
```
Comment (GabrielPiccini) [inline app/helpers/sessions_controller_helper.rb:1]: Same here

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```
Comment (GabrielPiccini) [inline app/helpers/sessions_helper.rb:1]: Same here

**Review by GabrielPiccini — CHANGES_REQUESTED** [review]
Changes

File: app/models/JwtBlackList.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class JWTBlacklist < ApplicationRecord
+    include Devise::JWT::RevocationStrategies::Blacklist
+  
+    self.table_name = 'jwt_blacklist'
+end
+
```
Comment (lucasenriquez) [inline app/models/JwtBlackList.rb:6]: Add a newline at the end of the file

File: db/migrate/CreateJwtBlacklist.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class CreateJwtBlacklist < ActiveRecord::Migration[5.0]
+    def change
+      create_table :jwt_blacklist do |t|
+        t.string :jti, null: false
+      end
+      add_index :jwt_blacklist, :jti
+    end
+end
+
```
Comment (lucasenriquez) [inline db/migrate/CreateJwtBlacklist.rb:9]: Same here

**Review by lucasenriquez — CHANGES_REQUESTED** [review]

**Review by lucasenriquez — APPROVED** [review]

**Pull request merged**